identity/validator_set.go

```diff
@@ -310,20 +362,25 @@ func (vs *ValidatorStore) GetEndBlockUpdate(ctx *ValidatorContext, req types.Req
 	minSelfDelegationAmount := stakingOptions.MinSelfDelegationAmount.BigInt().Int64()
 
 	if height > 1 || (len(vs.byzantine) > 0) {
+		// map for no top-power validators
```

Comment:
You mean non top-power validators?

Reply:
> You mean non top-power validators?

Yeah. That's the correct word and I'll correct it.